.github/MAINTAINERS.md

```diff
@@ -97,7 +97,6 @@ This will output a new key, associated with your email address to /tmp/new-ssh-k
 5. Update `./bin/split.sh` and `./.github/workflows/split-monorepo.yml` to include the new mirror and SSH key.
 6. Update the main README.md to include information about the new package.
 
-
```

Comment:
Fixed this in `main` a few mins ago https://github.com/readmeio/metrics-sdks/commit/32f8b76976a10a6c8ee27795d3edd999ee048f08

Reply:
ah, I'm too slow, thanks